inference-engine/src/mkldnn_plugin/ngraph_transformations/reshape_1d_ops.cpp

```diff
@@ -28,15 +29,38 @@ std::shared_ptr<ngraph::Node> convert(const ngraph::Output<ngraph::Node> & data,
     ngraph::Shape new_weights_shape(node->input_value(1).get_shape());
     new_weights_shape.insert(new_weights_shape.begin() + new_weights_shape.size() - 1, 1);
     auto weights = ngraph::op::util::reshapeTo(node->input_value(1), new_weights_shape);
+
     new_ops.push_back(weights);
 
-    return std::make_shared<T>(data,
-                               weights,
-                               new_strides,
-                               new_pads_begin,
-                               new_pad_end,
-                               new_dilations,
-                               node->get_auto_pad());
+    if (std::dynamic_pointer_cast<ngraph::op::TypeRelaxedBase>(node)) {
+        std::vector<ngraph::element::Type> inputPrecisions;
+        for (auto& inputs : node->inputs()) {
+            inputPrecisions.push_back(inputs.get_element_type());
+        }
+
+        std::vector<ngraph::element::Type> outputPrecisions;
+        for (auto& output : node->outputs()) {
+            outputPrecisions.push_back(output.get_element_type());
+        }
+
+        return std::make_shared<ngraph::op::TypeRelaxed<BaseOp>>(std::vector<ngraph::element::Type>{ngraph::element::f32, ngraph::element::f32},
```

Comment:
just didn't get why always fp32 here?

Reply:
it's temporary precisions to create Convolution, after creation these precisions will be changed on original